test/e2e/sync_test.go

```diff
@@ -0,0 +1,145 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	nodeclient "github.com/rollkit/rollkit/pkg/rpc/client"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFullNodeSyncsFromAggregator(t *testing.T) {
+	// scenario:
+	// start an aggregator and a fullnode
+	// when a tx for state update is executed
```

Comment:
When is a transaction ever sent in this test? Maybe we can send a tx to the aggregator and make sure the full node has synced it instead?

Reply:
ill add lazy and this one